### PR TITLE
allow fps lower then 31 to 60fps

### LINKED ISSUE #1963: Rename "60FPS player mode"
**Is your feature request related to a problem? Please describe.**

The "Alt+GreenFlag" looks strange, and the FPS can be changed (it's not necessarily 60)

**Describe the solution you'd like**

Change to "Alternate FPS player mode" or something similar

**Describe alternatives you've considered**

Remove "Alt+GreenFlag" from name, it's a little awkward to put how to use the addon in the name of the addon
For example the mute addon is not named "Ctrl+GreenFlag mute project"

### LINKED ISSUE #2778: Have projects move slower addon
### Is your request related to a problem? Please describe.

projects can only run at 30-240 fps using the 60fps addon. You can't make it run slower

### Describe the solution you'd like

A new addon like this

### Describe alternatives you've considered

remove the minimum for the 60fps addon

### Additional context

can this be done by just changing the min to 1, max to 29, and change the default?

### PR DESCRIPTION
**Pull request by Shluffy**
Resolves #2778 Fixes Slow-mo part of #2760 resolves #1963

### Changes
change the `addon.json` file to have the minimum be 1, changes description and changes name to be "custom fps player mode"
 
### Reason for changes
because this is just the opposite that makes sense to have
 
### Tests
I have not tested this.

### CODE REVIEW

**Comment by RedGuy12** [comment]
Changing the name and desc is fine IMO, just not the id.

**Comment by Shluffy** [comment]
All right, ty. I will change it

**Comment by RedGuy12** [comment]
Resolves #1963

**Comment by Shluffy** [comment]
@RedGuy12 is the name fine to change, just not the id?

**Comment by Shluffy** [comment]
is this going to be like this 4ever?

File: addons/60fps/addon.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Alt+GreenFlag 60FPS player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom FPS player mode",
+  "description": "Alt+Click the green flag to toggle Custom FPS.",
```
Comment (mxmou) [inline addons/60fps/addon.json:3]: I don't think "Custom FPS" should be capitalized.

File: addons/60fps/addon.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Alt+GreenFlag 60FPS player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom FPS player mode",
```
Comment (mxmou) [inline addons/60fps/addon.json:2]: "Player mode" should be changed to "project player mode" for consistency with the mute addon.

File: addons/60fps/addon.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+  "description": "Alt+Click the green flag to toggle Custom FPS.",
   "info": [
     {
       "type": "notice",
-      "text": "Projects will run twice as fast at 60FPS, because they typically run at 30FPS.",
+      "text": "Projects will run at a different speed when in use. Keep in mind that projects typically run at 30 fps",
```
Comment (mxmou) [inline addons/60fps/addon.json:7]: Notices should end with a dot, like descriptions. I would change it to "Enabling the custom FPS mode changes the project speed. Projects typically run at 30 FPS."

**Review by mxmou — CHANGES_REQUESTED** [review]

File: addons/60fps/addon.json (hunk 1 above)
Comment (Shluffy) [reply addons/60fps/addon.json:3]: fps was caps before, and custom is first word of name. Player and Mode should maybe be caps

**Comment by WorldLanguages** [comment]
> is this going to be like this 4ever?

I'll review this in a few days, I'm taking a little break

File: addons/60fps/addon.json (hunk 1 above)
Comment (CST1229) [reply addons/60fps/addon.json:3]: I think he was referring to the description.

Comment (Shluffy) [reply addons/60fps/addon.json:3]: @mxmou were you? i can change that

Comment (mxmou) [reply addons/60fps/addon.json:3]: > I think he was referring to the description.

I was.

Comment (Shluffy) [reply addons/60fps/addon.json]: i made the `C` in custom lowercase, but the fps was caps in the notice as well

**Comment by WorldLanguages** [comment]
IMO slow down / go faster buttons would be very useful, but they should be part of another addon. I don't think letting users enter less than 30 FPS on this addon makes sense. It would fix the problem temporarily, but don't forget this addon is enabled by default and 99% of its users will keep it at fixed 60

**Comment by WorldLanguages** [comment]
If you want to achieve this currently, you can run this in the settings page:

```js
vue.addonSettings["60fps"].framerate = 19
vue.updateSettings(vue.manifestsById["60fps"])
```

**Comment by return-misaPuding** [comment]
This is a useful feature that some people experimenting with and debugging some enhanced scratch projects could use and won´t harm other users (because as you already said, almost all of them (even me) don't even change the FPS and the ones which do so just won't set it to FPS lower than 30 unless they want it for some reason)

**Pull request closed without merging**